### PR TITLE
DA-783 add biobank oder to history on insert.

### PR DESCRIPTION
**Pull request by Seeker1911**
Add biobank order to history on initial insert
Add test for new orders inserted to history

### CODE REVIEW

File: rest-api/test/unit_test/api_test/biobank_order_api_test.py [hunk 1]
```diff
@@ -374,6 +375,16 @@ def test_insert_new_order(self):
...
 
+  def test_biobank_history_on_insert(self):
+    with self.bio_dao.session() as session:
+      self.summary_dao.insert(self.participant_summary(self.participant))
+      order_json = load_biobank_order_json(self.participant.participantId,
+                                           filename='biobank_order_2.json')
+      result = self.send_post(self.path, order_json)
+      load_biobank_order_json(self.participant.participantId, filename='biobank_order_1.json')
+      order_history = session.query(BiobankOrderHistory).first()
+      self.assertEqual(result['id'], order_history.biobankOrderId)
```
Comment (danqrodney) [inline rest-api/test/unit_test/api_test/biobank_order_api_test.py]: Also check version?

Can we also check to make sure the identifiers and samples got inserted into their history tables?

Check that this history record sticks around after an update?

File: rest-api/dao/biobank_order_dao.py [hunk 2]
```diff
@@ -124,6 +124,9 @@ def insert_with_session(self, session, obj):
     inserted_obj = super(BiobankOrderDao, self).insert_with_session(session, obj)
     ParticipantDao().add_missing_hpo_from_site(
         session, inserted_obj.participantId, inserted_obj.collectedSiteId)
+    self._update_history(session, obj)
```
Comment (danqrodney) [inline rest-api/dao/biobank_order_dao.py]: These methods get called in three different places, and always get called together. Consider combining them? (Or having a helper method that calls all three which gets invoked in the three places?)

File: rest-api/test/unit_test/api_test/biobank_order_api_test.py (hunk 1 above)
Comment (Seeker1911) [reply rest-api/test/unit_test/api_test/biobank_order_api_test.py]: done

File: rest-api/dao/biobank_order_dao.py (hunk 2 above)
Comment (Seeker1911) [reply rest-api/dao/biobank_order_dao.py]: done

**Pull request merged**